impeller/entity/geometry/vertices_geometry.cc

```diff
@@ -243,6 +244,64 @@ GeometryResult VerticesGeometry::GetPositionUVBuffer(
   };
 }
 
+GeometryResult VerticesGeometry::GetPositionUVColorBuffer(
+    Rect texture_coverage,
+    Matrix effect_transform,
+    const ContentContext& renderer,
+    const Entity& entity,
+    RenderPass& pass) const {
+  using VS = PorterDuffBlendPipeline::VertexShader;
+
+  auto index_count = indices_.size();
+  auto vertex_count = vertices_.size();
+  auto uv_transform =
+      texture_coverage.GetNormalizingTransform() * effect_transform;
+  auto has_texture_coordinates = HasTextureCoordinates();
+
+  size_t total_vtx_bytes = vertices_.size() * sizeof(VS::PerVertexData);
+  size_t total_idx_bytes = index_count * sizeof(uint16_t);
```

Comment:
Can you move this down to where it's used please?

Reply:
Done